src/absences/index.ts

```diff
@@ -1,12 +1,17 @@
 import BaseApi from '../baseApi';
-import responseHandler from '../utils/responseHandlers';
 import { Absence } from './types';
+import responseHandler from '../utils/responseHandlers';
+import { ReadOptions } from '../utils/types';
+import { AxiosRequestConfig } from 'axios';
 
 export default class Absences extends BaseApi {
   static resourceName = 'absences';
 
-  public read(): Promise<Absence[]> {
-    const response = this.get<Absence[]>(`${Absences.resourceName}/read`);
+  public read(options: ReadOptions = {}): Promise<Absence[]> {
```

Comment:
Should be readAll to be consistent with other endpoints.

Reply:
I thought we switched from `readAll`s to `read`s in #2 ?

Reply:
Rename single fetch (read/{id}) to **readById** (returns only one entity) and **readAll** should be just **read** with options/params (returns array of entities).